docs/en_US/NAS/ModelEvaluators.rst

```diff
@@ -27,17 +50,85 @@ Below is an example, ``transforms.Compose``, ``transforms.Normalize``, and ``MNI
   transform = serialize(transforms.Compose, [serialize(transforms.ToTensor()), serialize(transforms.Normalize, (0.1307,), (0.3081,))])
   train_dataset = serialize(MNIST, root='data/mnist', train=True, download=True, transform=transform)
   test_dataset = serialize(MNIST, root='data/mnist', train=False, download=True, transform=transform)
```

Comment:
it is a little confusion that we use both `nni.trace` and `serialize`, what is the difference between them?

Reply:
Oops, forgot to change `serialize` to `nni.trace`.
They are the same. I deprecated the `serialize` API (but it can still be used for backward compatibility).